e2e/src/comics.e2e-spec.ts

```diff
@@ -30,6 +30,12 @@ describe('Comics', () => {
     );
   });
 
+  it('should use correct style for currently selected comic', async () => {
+      expect(
+      await element(by.css('img.responsive-img.current')).isDisplayed()
```

Comment:
`isDisplayed` is already a statement so I don't think we need it to be wrapped in an `expect` here. Just have this line on its own

Reply:
Happy to do this.  I followed the same pattern for collections spec on line 28 where I did the single line, and you asked for it to have an assertion.  Can this be different for a reason?